rollup/main/default/objects/Rollup__mdt/fields/GrandparentRelationshipFieldPath__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>GrandparentRelationshipFieldPath__c</fullName>
```

Comment:
I think I'll stick to it for now, but this is good food for thought in the pre-managed package era. Since the next big thing I want to do is re-do the folder structure, I'll 📍this - I love the idea, overall, and think I'll probably update them long-term